src/pages/Swap/Components/Tabs.tsx

```diff
@@ -85,21 +86,21 @@ export function Tabs() {
       <TabsRow>
         <Button
           onClick={() => {
-            setActiveTab(SwapTab.Swap)
+            setActiveTab(SwapTab.SWAP)
           }}
-          className={activeTab === SwapTab.Swap ? 'active' : ''}
+          className={activeTab === SwapTab.SWAP ? 'active' : ''}
```

Comment:
I guess you could have done this: `className={activeTab === SwapTab.SWAP && 'active'}`. This syntax is a common thing in **styled-components**, too. Seems a bit cleaner to me. 😃

Reply:
I know this can work in JS, but it seems that TS doesn't like it. Because you can pass `false` to a `string | undefined` type prop.